.github/workflows/scripts/pre-release/references.sh

```diff
@@ -32,3 +33,19 @@ if [[ "$results" != "" ]]; then
     exit 1
 fi
 
+# Verify documentation refers to the most recent release tag
+# TODO(https://github.com/slsa-framework/slsa-github-generator/milestone/3):
```

Comment:
Maybe we should add something like `-not -path "./internal/builders/container/*"` to the find command instead of allowing `main`?

Also, let's create & link to an issue rather than the milestone.

Reply:
On the `-not -path`: this wouldn't remove the alert on the `detect-workflow` workflow, though that might be solved by what you suggested in the other comment.

On the issue: would #409 be alright? I assume it'll only be considered closed when the container builder is actually released.

Reply:
> On the `-not -path`: this wouldn't remove the alert on the `detect-workflow` workflow, though that might be solved by what you suggested in the other comment.

IIUC, It should only exclude the `./internal/builders/container/` directory which contains the generic container builder's code and docs. I don't think `detect-workflow` would be affected.

> On the issue: would #409 be alright? I assume it'll only be considered closed when the container builder is actually released.

SG